doc/user/content/get-started/quickstart.md

```diff
@@ -12,297 +12,764 @@ aliases:
+
+   *Optional exercise*. You can issue the `SUBSCRIBE` statement without the
+   `WITH (snapshot = false)` to display both the flippers that exist at the
+   start of the command as well as display new flippers as they come in. If the
+   results are paginated, you may need to go to the end to see the new flippers.
 
-1. Pick one of the buyers from the list maintained in the window that is running
-the `SUBSCRIBE`, and mark them as fraudulent by adding them to the
-`fraud_accounts` table.
+## Step 6. Include a table for known flippers
+
+Currently, flippers are determined by the auction activities data being
+auto-generated. Assume that, separate from your auction data, you receive
+independent notifice that certain users are flippers. Instead of waiting for
+their auction data to flag them as flippers, you want to immediately store this
+data and incorporate the table in flagging flippers.
+
+1. Use [`CREATE TABLE`](/sql/create-table) to create a `known_flippers` table
+   that can be manually populated with known flippers.
 
    ```mzsql
-   INSERT INTO fraud_accounts VALUES (<id>);
+   CREATE TABLE known_flippers (flipper_id bigint);
    ```
 
-   This should cause the flagged buyer to immediately drop out of the Top 5! If
-   you click **Show diffs**, you'll notice that the picked buyer was kicked
-   out, and the next non-fraudulent buyer in line automatically entered the
-   Top 5.
+1. Drop the existing `flippers` view and recreate using both the
+   `flipper_activities` and the new `known_flippers` table.
+
+   1. Use the [`DROP VIEW`](/sql/drop-view) command to drop the existing
+      `flippers` view.
+
+      ```mzsql
+       DROP VIEW flippers;
+      ```
+
+   1. Recreate the view to include the `known_flippers` table:
 
-   When you're done, cancel out of the `SUBSCRIBE` using **Stop streaming**, and close
-   the secondary browser window.
+      ```mzsql
+      CREATE VIEW flippers AS
+      SELECT flipper_id
+      FROM (
+          SELECT flipper_id
+          FROM flip_activities
+          GROUP BY flipper_id
+          HAVING count(*) >= 2
 
-## Step 4. Serve correct results
+          UNION ALL
 
-With fraud out of the way, you can now shift your focus to a different
-operational use case: profit & loss alerts.
+          SELECT flipper_id
+          FROM known_flippers
+      );
+      ```
 
-**Operational work needs to act on correct and consistent data.** Before you
-  warn a user that they've spent much more than they've earned, you want to be
-  sure your results are trustworthy — it's real money we're talking about,
-  after all!
+   1. Subscribe to `flippers`.
 
-1. Create a view to track the sales and purchases of each auction house user.
+      ```mzsql
+      SUBSCRIBE TO (
+        SELECT *
+        FROM flippers
+      ) WITH (snapshot = false)
+      ;
+      ```
+
+1. In another [Materialize console](https://console.materialize.com/), insert
+   new data into the `known_flippers` table .
```

Comment:
Oof. We had special tech for doing table insertions because the context shift was annoying. Should we repurpose that here, or use multiple browser windows?

Reply:
Oh ... I wasn't aware of the special tech for table insertion.  We can definitely can get rid of the multiple windows if the console allows it.

Reply:
Updated in patch.